src/api/firebase.js

```diff
@@ -139,21 +139,26 @@ export async function createList(userId, userEmail, listName) {
 export async function shareList(listPath, currentUserId, recipientEmail) {
 	// Check if current user is owner.
 	if (!listPath.includes(currentUserId)) {
-		return;
+		return 'match';
 	}
 	// Get the document for the recipient user.
 	const usersCollectionRef = collection(db, 'users');
 	const recipientDoc = await getDoc(doc(usersCollectionRef, recipientEmail));
 	// If the recipient user doesn't exist, we can't share the list.
 	if (!recipientDoc.exists()) {
-		return;
+		return false;
```

Comment:
Non blocking comment:

This feels redundant to me. 

If you keep it as it was before with just the `return;` the value returned will be `undefined` in which it would be a falsy value anyway and hit the `else` statement (line 69 in src/views/ManageList.jsx) as expected.

Reply:
removed redundant false from the return statement!